addons/mail/static/src/core/common/message_reaction_list.xml

```diff
@@ -10,7 +10,7 @@
     </t>
     
     <t t-name="mail.MessageReactionList.button">
-        <button class="position-relative o-mail-MessageReaction btn d-flex p-0 border rounded-2 mb-1 fs-5 px-1 gap-1 align-items-center" t-on-click="() => this.onClickReaction(props.reaction)" t-on-contextmenu="onContextMenu" t-on-mouseenter="loadEmoji" t-ref="reactionButton" t-att-class="{
+        <button class="position-relative o-mail-MessageReaction btn d-flex p-0 border rounded-2 mb-1 fs-5 px-1 gap-1 align-items-center" t-att-disabled="!props.message.canAddReaction()" t-on-click="() => this.onClickReaction(props.reaction)" t-on-contextmenu="onContextMenu" t-on-mouseenter="loadEmoji" t-ref="reactionButton" t-att-class="{
```

Comment:
The disabled here adds a strongly greyed-out to the reactions, making them a bit unreadable.
![image](https://github.com/user-attachments/assets/36878d6d-e9bc-4c6b-b352-889f0b4f3707)

I think it would be best to keep the style intact here, either by removing the disabled style on this button, or by not disabling it but making sure the click handler does nothing.

The disable also removes the contextmenu apparently, which is not something we need to prevent.

Reply:
Agree, keep the static style of the button, just no hover effect and click on button does nothing, but click on tooltip must show the reaction menu.

Reply:
Hello @seb-odoo  @alexkuhn 
The changes are complete.

As for the hover effect, it seems that there is only one hover effect applied when `hasSelfReacted` is true via `button > .text-primary:hover`, which darkens the purple shade slightly to the text. To handle this, we might need to implement some custom code, but I don't think it's necessary since the effect is barely noticeable in the UI. What do you suggest? If I'm overlooking something, please point it out.

Reply:
@alexkuhn 
That's correct. It's done.